pkg/wallet/account/account.go

```diff
@@ -0,0 +1,265 @@
+package account
+
+import (
+	"crypto/ed25519"
+	"crypto/rand"
+	"fmt"
+
+	"github.com/awnumar/memguard"
+	"github.com/btcsuite/btcutil/base58"
+	"github.com/massalabs/station-massa-wallet/pkg/crypto"
+	"github.com/massalabs/station-massa-wallet/pkg/types"
+	"github.com/massalabs/station-massa-wallet/pkg/types/object"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	AccountLastVersion = 1
+)
+
+type Account struct {
+	Version      *uint8                    `yaml:"Version"`
```

Comment:
Why is it a pointer ?

Reply:
idk. I need help on the version

Reply:
I need it to be a pointer because it's a mandatory field in the yaml, and the yaml unmarshal will put value nil and so that I can throw an error if the value is nil after unmarshaling the yaml

Reply:
Your reason seems shaky: using uint8(0) as UnknownVersion value would avoid using a pointer and will be consistent with the default value of an uint8.

Reply:
yes but value 0 for a version is a correct value, our standard wallet file format first version is 0.

Reply:
let's take 0 as unknown version

`UnknownVersion := uint8(0)`